cmd/cape/cmd/delete.go

```diff
@@ -17,7 +17,7 @@ var deleteCmd = &cobra.Command{
 	Long:  "Delete a deployed function, takes function id.\n",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		err := delete(cmd, args)
-		if _, ok := err.(UserError); !ok {
+		if _, ok := err.(UserError); ok {
```

Comment:
I don't think you want to flip this. The idea is that if it is a user error, then we should print the usage information. This will hide the usage info if it is a user error

Reply:
Okay

Reply:
Suppressing usage information when it is a UserError goes against the norm we established for other commands.

Reply:
I think if the user makes a mistake, we should give user info. e.g. the pass an invalid ID. If something goes wrong outside of their control, we should just show the error